modules/040-node-manager/openapi/values.yaml

```diff
@@ -264,3 +266,15 @@ properties:
                 minPerZone: 1
                 zones:
                   - zonea
+      upmeterDiscovery:
+        type: object
+        required:
+          - ephemeralNodeGroupNames
```

Comment:
There is a default value, why do you require it?

Reply:
The same for nginx-ingress module.

Reply:
I think you are right. I even have already guarded empty values im configmaps templates. 

I know what i missed out. I treat the schema as JSON schema. And`default` annotation does nothing by its spec. At all. It only documents the behavior that is not provided by schema processor itself. https://json-schema.org/understanding-json-schema/reference/generic.html#id2

But we use OpenAPI processor, which applies defaults in-place. 

We need neither of these.

Reply:
So I removed them